js/repeater.js

```diff
@@ -76,7 +76,8 @@
 		this.$pageSize.selectlist();
 		this.$primaryPaging.find('.combobox').combobox();
 		this.$search.search({
-			searchOnKeyPress: this.options.searchOnKeyPress
+			searchOnKeyPress: this.options.searchOnKeyPress,
+			allowCancel: true
```

Comment:
This means that the search in the repeater will _always_ be cancelable.  Should we also support overriding this option on repeater for someone who does _not_ want repeater searches to be cancelable?